<!--
-->
<launch>
  <node pkg="move_base" type="move_base" respawn="true" name="move_base" output="screen" clear_params="true">
  <!--<node pkg="move_base" type="move_base" respawn="true" name="move_base" output="screen">-->
    <rosparam file="$(find obrobot_navigation)/config/odom/mapping/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find obrobot_navigation)/config/odom/mapping/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find obrobot_navigation)/config/odom/mapping/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find obrobot_navigation)/config/odom/mapping/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find obrobot_navigation)/config/odom/mapping/base_global_planner_param.yaml" command="load" />

    <rosparam file="$(find obrobot_navigation)/config/odom/mapping/teb_local_planner_params.yaml" command="load" />
    <rosparam file="$(find obrobot_navigation)/config/odom/mapping/move_base_params.yaml" command="load" />

    <!--<rosparam file="$(find obrobot_navigation)/config/odom/mapping/costmap_conversion_params.yaml" command="load" />-->
    <!--<param name="base_global_planner" value="global_planner/GlobalPlanner"/> -->
    <!--<param name="base_global_planner" value="kplanner/KplannerROS"/> -->
    <!--<param name="planner_frequency" value="1.0" />
    <param name="planner_patience" value="5.0" />

    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
    <param name="controller_frequency" value="5.0" />
    <param name="controller_patience" value="15.0" />-->

  </node>
  
</launch>
